code/game/machinery/OpTable.dm

```diff
@@ -100,7 +100,7 @@
 	if(!anes_tank)
 		to_chat(user, SPAN_WARNING("There is no anesthetic tank connected to the table, load one first."))
 		return
-	if(H.wear_mask && !H.drop_inv_item_on_ground(H.wear_mask))
+	if(H.wear_mask && NODROP)
```

Comment:
This is just checking a random bit flag. It needs to check if NODROP is active in the mask's flags

Reply:
outdated version >:)